src/graphql/operations/votes.ts

```diff
@@ -56,6 +56,39 @@ async function query(parent, args, context?, info?) {
     return Promise.reject('request failed');
   }
 
+  // filter by choice
+  if (where.choice) {
+    votes = votes.filter(v => {
+      if (typeof v.choice === 'number') {
+        return v.choice === where.choice;
+      }
+      if (typeof v.choice === 'object') {
+        if (Array.isArray(v.choice)) {
+          return v.choice.includes(where.choice);
+        } else {
+          return Object.keys(v.choice).includes(where.choice.toString());
+        }
+      }
+      return false;
+    });
```

Comment:
This may affect `first` and `skip` for pagination right?

Reply:
Maybe we should have a new input variable `filter` instead of `where`

Reply:
You mean as a clear way of saying "this filtering happens after pagination"?

For a moment I thought about doing the pagination in JS too. Removing the `LIMIT` from the query, at least when there's a choice filter. But that doesn't feel right.

So yea... idk. We just tell people it's not possible and they have to do the filtering themselves, in their client?

Reply:
@mktcode Yes, they can filter on their side, we would implement this when we have an elegant way to do it. Who is asking this feature?